packages/2018-transportation-systems/src/components/DeclineInRidership/index.js

```diff
@@ -1,25 +1,93 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, LineChart } from '@hackoregon/component-library';
+
+import { fetchRidershipOverTime } from '../../state/decline-in-ridership/actions';
+import {
+  isRidershipOverTimePending,
+  getRidershipOverTimeData,
+} from '../../state/decline-in-ridership/selectors';
+
+const sampleSimpleData = [
+  { x: 0, y: 20 },
+  { x: 10, y: 30 },
+  { x: 20, y: 50 },
+  { x: 30, y: 40 },
+];
```

Comment:
Looks like this sample data is no longer being used.

Reply:
Yep, removed.